src/controllers/drivers.cr

```diff
@@ -33,7 +33,22 @@ module PlaceOS::Api
     end
 
     def show
-      render json: current_driver
+      driver = current_driver
+      include_compilation_status = !params.has_key?("compilation_status") || params["compilation_status"] != "false"
+
+      if include_compilation_status
+        file_name = driver.file_name.as(String)
+        commit = driver.commit.as(String)
+        repository = driver.repository.as(String)
+        tag = driver.id.as(String)
+        status = Api::Drivers.compilation_status(file_name, commit, repository, tag, logger.request_id)
+        response = with_fields(driver, {
+          :compilation_status => status,
+        })
+        render json: response
+      else
```

Comment:
this else isn't required as `render` returns

Reply:
thought i'd make it explicit, but can amend.